source/extensions/quic_listeners/quiche/quic_filter_manager_connection_impl.h

```diff
@@ -120,6 +117,8 @@ class QuicFilterManagerConnectionImpl : public Network::ConnectionImplBase {
   // Called when aggregated buffered bytes across all the streams declines to low watermark.
   void onSendBufferLowWatermark();
 
+  static Buffer::Instance& emptyBuffer() { MUTABLE_CONSTRUCT_ON_FIRST_USE(Buffer::OwnedImpl); }
```

Comment:
Sorry I think I led your astray here. I didn't realize this is mutable. If mutable this seems not great so probably revert this? cc @danzh2010

Reply:
Sure, reverted back.